DUnitX.Assert.pas

```diff
@@ -359,6 +359,23 @@ class procedure Assert.AreEqual(const expected, actual: boolean; const message:
     FailFmt(SUnexpectedErrorStr ,[BoolToStr(expected, true), BoolToStr(actual, true), message], ReturnAddress);
 end;
 
+class procedure Assert.AreEqual(const expected, actual: string; const compformat: TDUnitXComparableFormatClass; const ignoreCase: boolean; const message: string);
+begin
+  DoAssert;
+  if ignoreCase then
+  begin
+    if not SameText(expected,actual) then
+      FailStrCompare(expected, actual, compformat, message, ReturnAddress);
```

Comment:
The caller can pass the test even if their data does not match the supplied formatter requirements.